src/main/java/de/progen_bot/util/Settings.java

```diff
@@ -36,13 +36,12 @@ private Settings() {
     }
 
     //#region Bot
-    public static final List<String> BOT_OWNERS =   Collections.unmodifiableList(GSON.fromJson(get(BOT_KEY, "owners").getAsJsonArray(), List.class));
+    public static final List<String> BOT_OWNERS =   Arrays.asList(GSON.fromJson(get(BOT_KEY, "owners").getAsJsonArray(), String[].class));
     public static final String PREFIX           =   get(BOT_KEY, "prefix").getAsString();
     public static final String TOKEN            =   get(BOT_KEY, "token").getAsString();
     public static final String API_PORT         =   String.valueOf(get(BOT_KEY, "apiPort").getAsInt());
 
-    @SuppressWarnings("unchecked")
-    public static final List<String> MUSIC      =   Collections.unmodifiableList(GSON.fromJson(get(BOT_KEY, "music").getAsJsonArray(), List.class));
+    public static final List<String> MUSIC      =   Arrays.asList(GSON.fromJson(get(BOT_KEY, "music").getAsJsonArray(), String[].class));
```

Comment:
Lists should be unmodifiable.

Reply:
Yeah true. Thought its done with final but thats only for initialization.